regulation-worker/internal/client/client.go

```diff
@@ -45,12 +43,14 @@ func (j *JobAPI) Get(ctx context.Context) (model.Job, error) {
 	req.SetBasicAuth(j.WorkspaceToken, "")
 	req.Header.Set("Content-Type", "application/json")
 
-	pkgLogger.Debugf("making request: %v", req)
 	resp, err := j.Client.Do(req)
-	if err != nil {
-		pkgLogger.Errorf("http request failed with error: %v", err)
+	if err, ok := err.(net.Error); ok && err.Timeout() {
```

Comment:
`os.IsTimeout(err)` captures this and much more, including `context.DeadlineExceeded`
```suggestion
	if os.IsTimeout(err) {
```

Reply:
done